pallets/asset-index/src/mock.rs

```diff
@@ -169,7 +169,7 @@ impl primitives::traits::RedemptionFee<BlockNumber, Balance> for RedemptionFee {
 }
 
 impl pallet_asset_index::Config for Test {
-	type AdminOrigin = frame_system::EnsureSignedBy<AdminAccountId, AccountId>;
```

Comment:
We don't need to restrict `EnsureSignedBy` in tests since this is only work for tests